draft-barnes-acme.md

```diff
@@ -133,9 +133,6 @@ Subject Public Key:
 Account Key Pair:
 : A key pair for which the ACME server considers the holder of the private key authorized to manage certificates for a given identifier.  The same key pair may be authorized for multiple identifiers.
 
-Recovery Token:
```

Comment:
HMAC? If you're relying on preimage resistance, why not just send H(token)

On Wed, Jun 24, 2015 at 6:46 PM, Peter Eckersley <notifications@github.com>
wrote:

> In draft-barnes-acme.md
> <https://github.com/letsencrypt/acme-spec/pull/156#discussion_r33215820>:
>
> > @@ -133,9 +133,6 @@ Subject Public Key:
> >  Account Key Pair:
> >  : A key pair for which the ACME server considers the holder of the private key authorized to manage certificates for a given identifier.  The same key pair may be authorized for multiple identifiers.
> >
> > -Recovery Token:
>
> We should not remove the Recovery Token; let's have the client send an
> HMAC of one, which the server can record.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/letsencrypt/acme-spec/pull/156/files#r33215820>.
>